test/tests/parallel/set1/config_overrides.bats

```diff
@@ -15,6 +15,9 @@ function setup_file() {
 
 function teardown_file() { _default_teardown ; }
 
+# The `postconf` command can query both `main.cf` and `master.cf` at `/etc/postfix/`.
+# Reference: http://www.postfix.org/postconf.1.html
```

Comment:
Maintainer docs reference for this test file, since there's a few different options used with the `postconf` command in the test cases below, probably helpful to have here 😅